README.md

```diff
@@ -258,9 +258,8 @@ The integration tests helps the validation for resources interacting with third
 
 ### Autoclose stale issues and PRs
 
-- After 30 days of no activity (no comments or commits are on an issue or PR) we automatically tag it as “stale” and add a message: "This issue has gone 30 days without any activity and meets the project’s definition of ‘stale’. This will be auto-closed if there is no new activity over the next 60 days. If the issue is still relevant and active, you can simply comment with a “bump” to keep it open, or add the “[Status] Not Stale” label. Thanks for keeping our repository healthy!"
-
-- After 60 more days of no activity we automatically close the issue / PR.
+- After 30 days of no activity (no comments or commits on an issue/PR) we automatically tag it as "stale" and add a message: ```This issue/PR has gone 30 days without any activity and meets the project's definition of "stale". This will be auto-closed if there is no new activity over the next 60 days. If the issue is still relevant and active, you can simply comment with a "bump" to keep it open, or add the label "not_stale". Thanks for keeping our repository healthy!```
+- After 60 more days of no activity we automatically close the issue/PR.
```

Comment:
keeping this consistent with the CLI and go client https://github.com/mongodb/mongodb-atlas-cli/pull/1490/files